src/jit/emitxarch.cpp

```diff
@@ -1925,7 +1925,7 @@ UNATIVE_OFFSET      emitter::emitInsSizeAM(instrDesc* id, size_t code)
         // Most 16-bit operands will require a size prefix .
         // This refers to 66h size prefix override.
 
-        if  (    (attrSize == EA_2BYTE)
+        if  (    attrSize == EA_2BYTE
```

Comment:
Be that as it may, it is in violation of our coding conventions.  If it must be changed, it should be changed to include that line in the #if.  Same for the others below.